packages/@lwc/ssr-compiler/src/compile-template/component.ts

```diff
@@ -75,22 +77,36 @@ function reflectAriaPropsAsAttrs(props: IrProperty[]): IrAttribute[] {
         .filter((el): el is NonNullable<IrAttribute> => el !== null);
 }
 
-export const Component: Transformer<IrComponent> = function Component(node, cxt) {
-    // Import the custom component's generateMarkup export.
-    const childGeneratorLocalName = `generateMarkup_${toPropertyName(node.name)}`;
-    const importPath = kebabcaseToCamelcase(node.name);
-    const componentImport = bImportGenerateMarkup(childGeneratorLocalName, importPath);
-    cxt.hoist(componentImport, childGeneratorLocalName);
-    const childTagName = node.name;
-
+export function ImportedComponent(
```

Comment:
Could you add a comment here to disambiguate `ImportedComponent` from `Component`?